pkg/webhooks/leaderworkerset_webhook.go

```diff
@@ -51,14 +51,6 @@ var _ webhook.CustomDefaulter = &LeaderWorkerSetWebhook{}
 func (r *LeaderWorkerSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	lws := obj.(*v1.LeaderWorkerSet)
 
-	if lws.Spec.LeaderWorkerTemplate.RestartPolicy == "" {
-		lws.Spec.LeaderWorkerTemplate.RestartPolicy = v1.DefaultRestartPolicy
-	}
-
-	if lws.Spec.RolloutStrategy.Type == "" {
-		lws.Spec.RolloutStrategy.Type = v1.RollingUpdateStrategyType
-	}
-
 	if lws.Spec.RolloutStrategy.Type == v1.RollingUpdateStrategyType && lws.Spec.RolloutStrategy.RollingUpdateConfiguration == nil {
```

Comment:
I mean here lws.Spec.RolloutStrategy.Type might be empty if we didn't set the value.